ZipBuilder/Package.swift

```diff
@@ -26,7 +26,7 @@ let package = Package(
     .executable(name: "UpdateFirebasePod", targets: ["UpdateFirebasePod"]),
   ],
   dependencies: [
-    .package(url: "https://github.com/apple/swift-protobuf.git", .exact("1.2.0")),
+    .package(url: "https://github.com/apple/swift-protobuf.git", .exact("1.7.0")),
```

Comment:
There's a recommendation that the protobufs used are generated with the same version of `swift-protobuf` tool. Do you mind checking to make sure there are no source changes?

(I realize now this should be in a comment above the dependency)

Reply:
Updated comment. 

It turns out that the last proto update in #4454 was already with 1.7.0:
`Warning: swift-protobuf 1.7.0 is already installed and up-to-date`

Reply:
Ahh got it, thanks!